superset/assets/javascripts/SqlLab/components/App.jsx

```diff
@@ -32,11 +32,16 @@ class App extends React.PureComponent {
     this.setState({ hash: window.location.hash });
   }
   getHeight() {
-    const navHeight = 90;
-    const headerHeight = $('.nav-tabs').outerHeight() ?
-      $('.nav-tabs').outerHeight() : $('#search-header').outerHeight();
-    const warningHeight = $('#navbar-warning').outerHeight();
-    const alertHeight = $('#sqllab-alerts').outerHeight();
+    const warningEl = $('#navbar-warning');
```

Comment:
all of these elements wouldn't work in a single div. i think it makes more sense to be explicit about what elements we are including in this calculation, rather than a using single class to mark them in the templates.